src/core/js/adapt.js

```diff
@@ -106,9 +106,11 @@ define([
         }
 
         var navigationHeight = $(".navigation").outerHeight();
+        // prevent scroll issue when component description aria-label coincident with top of component
+        var ariaLabelBuffer = $(selector.is('.component')) ? 5 : 0;
```

Comment:
Yes, it's unusual, but it seems IE8 doesn't have this problem, just IE9 and above!